pkg/controller/backup_controller.go

```diff
@@ -230,6 +234,22 @@ func (b *backupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	switch original.Status.Phase {
 	case "", velerov1api.BackupPhaseNew:
 		// only process new backups
+	case velerov1api.BackupPhaseInProgress:
+		// if backup is in progress, we should not process it again
+		// we want to mark it as failed to avoid it being stuck in progress
+		// if this backupReconciler uuid is the same as the one in the backup, then we are the one who is processing it
+		// if so, mark it as failed, last loop did not successfully complete the backup
+		if original.Annotations[velerov1api.BackupControllerUID] == string(b.uuid) {
+			log.Debug("Backup was processed by this backupReconciler and still has in progress status upon reconcile, marking it as failed")
+		}
+		failedCopy := original.DeepCopy()
```

Comment:
Shouldn't this be inside the block above -- only set failed if the if test is true? Currently, only the debug statement is conditionalized.